src/ExpressionEvaluator/CSharp/Source/ResultProvider/CSharpResultProvider.csproj

```diff
@@ -12,22 +12,14 @@
     <AuthenticodeCertificateName>MicrosoftSHA1Win8WinBlue</AuthenticodeCertificateName>
     <RootNamespace>Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator</RootNamespace>
     <AssemblyName>Microsoft.CodeAnalysis.CSharp.ExpressionEvaluator.ResultProvider</AssemblyName>
-    <!--
-      As a convenience for developer machines without .NET 2.0 installed, we'll allow building against v4.*.
-      However, "real" builds must still target v2.0, and we always want to target 2.0 if it's present.
-    -->
-    <TargetFrameworkVersion Condition="Exists('$(WinDir)\Microsoft.NET\Framework\v2.0.50727\mscorlib.dll') OR '$(RealSignBuild)' == 'true'">v2.0</TargetFrameworkVersion>
-    <!--
-      We get warnings about missing XML comments for publicly visible members in Microsoft.CodeAnalysis.CSharp.SyntaxKind
-    -->
-    <NoWarn>1591</NoWarn>
+    <TargetFrameworkVersion>v4.5</TargetFrameworkVersion>
```

Comment:
Yes, this is only the version of tools that are used. The properties set below make sure that msbuild doesn't try to apply implicitly generated stuff for 4.5.